Lib/sqlite3/__main__.py

```diff
@@ -0,0 +1,91 @@
+import sqlite3
+import sys
+
+from argparse import ArgumentParser
+from code import InteractiveConsole
+from textwrap import dedent
+
+
+class SqliteInteractiveConsole(InteractiveConsole):
+
+    def __init__(self, database):
+        super().__init__()
+        self._con = sqlite3.connect(database, isolation_level=None)
+        self._cur = self._con.cursor()
+
+    def __del__(self):
+        self._con.close()
+
+    def runsql(self, sql):
+        if sqlite3.complete_statement(sql):
```

Comment:
Would not be better to negate the condition and reduce the indentation of the main code?
```
        if not sqlite3.complete_statement(sql):
            return True

```